<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messages</class>
 <widget class="QDialog" name="Messages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Messages{
	background-image: url(:/ms.jpg);
}
#frame{
 background-color: rgba(255, 255, 255, 0.25); 
border-radius: 35px;
}
#lineEdit{
background-color:rgba(82, 82, 82,0.5);
color:white;
}

#messageButton {
color:black;}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>451</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="messageButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
      <pointsize>18</pointsize>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="text">
     <string>MESSAGES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Enter your message</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
